pkg/controller/networkpolicy/networkpolicy_controller_perf_test.go

```diff
@@ -54,11 +56,11 @@ The metrics are not accurate under the race detector, and will be skipped when t
 */
 func TestInitXLargeScaleWithSmallNamespaces(t *testing.T) {
 	namespaces, networkPolicies, pods := getXLargeScaleWithSmallNamespaces()
-	testComputeNetworkPolicy(t, 10*time.Second, namespaces, networkPolicies, pods)
+	testComputeNetworkPolicy(t, 10*time.Second, namespaces, networkPolicies, nil, pods, nil)
 }
 
 func getXLargeScaleWithSmallNamespaces() ([]*corev1.Namespace, []*networkingv1.NetworkPolicy, []*corev1.Pod) {
-	getObjects := func() ([]*corev1.Namespace, []*networkingv1.NetworkPolicy, []*corev1.Pod) {
+	getObjects := func() ([]*corev1.Namespace, []*networkingv1.NetworkPolicy, []*v1alpha1.NetworkPolicy, []*corev1.Pod, []*v1alpha2.ExternalEntity) {
```

Comment:
You can change the function's signature to:
```
func() (namespaces []*corev1.Namespace, networkPolicies []runtime.Object, entities []runtime.Object)
```
Then the callers don't have to have the extra 2 return values.

Reply:
done.